src/ui/server.ts

```diff
@@ -171,6 +171,16 @@ app.param('eventId', asyncHandler(courseController.loadEvent))
 
 app.use('/courses/:courseId/:moduleId/xapi', asyncHandler(xApiController.proxy))
 
+/**
+ * The below handler is added as there are xapi calls done against learning-record which were not handled and were
+ * caught by lusca CSRF check - resulting with big number of error messages.
+ *
+ * As it hit 100% error rate the below handler is proposed to remediate the errors appearing - it is to be
+ * investigated whether the calls should be handled (or could they be removed completely).
+ */
+app.use('/learning-record/:learnerRecordId/:notHandledModuleId/xapi',
+	(req: express.Request, res: express.Response) => res.sendStatus(204))
```

Comment:
I can see response code is a happy one, and wondering if the request (failed CSRF check) is logged  somewhere? @michalpiatek